learners/setup.md

```diff
@@ -5,134 +5,130 @@ title: Setup
 
 ::::::::::::::::::::::::::::::::::::::: discussion
 
-### Installing Python using Anaconda
+### Installing Python
 
 [Python][python] is a popular language for scientific computing, and a frequent choice
-for machine learning as well. Installing all of its scientific packages
-individually can be a bit difficult, however, so we recommend the installer [Anaconda][anaconda]
-which includes most (but not all) of the software you will need.
-
-Regardless of how you choose to install it, please make sure you install Python
-version 3.x (e.g., 3.4 is fine). Also, please set up your python environment at
-least a day in advance of the workshop.  If you encounter problems with the
-installation procedure, ask your workshop organizers via e-mail for assistance so
+for machine learning as well. 
+To install Python, follow the [Beginner's Guide](https://wiki.python.org/moin/BeginnersGuide/Download) or head straight to the [download page](https://www.python.org/downloads/).
+
+Please set up your python environment at least a day in advance of the workshop.
+If you encounter problems with the installation procedure, ask your workshop organizers via e-mail for assistance so
 you are ready to go as soon as the workshop begins.
 
 :::::::::::::::::::::::::::::::::::::::::::::::::::
 
-:::::::::::::::: spoiler
-
-### Windows
+## Installing the required packages{#packages}
 
-Checkout the [video tutorial][video-windows] or:
+[Pip](https://pip.pypa.io/en/stable/) is the package management system built into Python.
+Pip should be available in your system once you installed Python successfully.
 
-1. Open [https://www.anaconda.com/products/distribution][anaconda-distribution]
-with your web browser.
-2. Download the Python 3 installer for Windows.
-3. Double-click the executable and install Python 3 using _MOST_ of the
-   default settings. The only exception is to check the
-   **Make Anaconda the default Python** option.
+1. Create a [virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#create-and-use-virtual-environments) called `dl_workshop`:
```

Comment:
I am not sure this is correct because the new folder is created automatically when the command is run:
```console
% ls dl_workshop
ls: dl_workshop: No such file or directory
% python3 -m venv dl_workshop
% ls dl_workshop             
bin		include		lib		pyvenv.cfg
```
Or am I missing something?

Reply:
The instructions as of 10b2482 worked fine on my freshly installed windows, which for some reason did not have python pre-installed.

Note that below it states "To start Jupyter Lab, open a terminal (Mac/Linux) or Anaconda prompt (Windows) and type the command". I did this in command prompt (as was part of the instructions above) and works fine too. It probably makes more sense to not mention anaconda prompt here, as there are no instructions to install anaconda on this site (even though it starts all the way from python installation.

Reply:
Oh, @carschno I wasn't aware of that! I did actually not try it without first creating the folder 🙈. Please ignore my suggestion.

I agree with @DaniBodor all references to Anaconda or Anaconda prompt need to be removed.

Reply:
Ah, spot-on, thanks! I have removed the remaining Anaconda references, and replace the explicit `bash` references (code blocks) with `shell`.
@DaniBodor do you know what the standard terminal/command line is on Windows so we could refer to that explicitly? On the other hand, we assume that people know the command line, so we might just leave it as it is.

BTW, I have checked the [Python beginner instructions](https://wiki.python.org/moin/BeginnersGuide/Download) for reference, and they do not go into the details of command line either:

> [...] you can try running the python command to start the Python interpreter to check and see if it is already installed. On windows you can try the py command which is a launcher which is more likely to work

So perhaps we can presume knowledge about how to run a command.

Reply:
I think this is clear enough:
`Open up a terminal (on Linux/macOs) or Command Prompt (on Windows) ` 
Better to be explicit I think.

That was also part of my initial suggestion. I suggest to keep that part (first step opening a terminal/command prompt) and toss the part on creating a new folder.